paper-toolbar.html

```diff
@@ -102,11 +104,11 @@
     }
 
     :host(.medium-tall) {
-      height: 128px;
+      height: var(--paper-toolbar-height, 128px);
```

Comment:
These changes are to work around
https://github.com/Polymer/polymer/issues/2531 :bug: